services/src/main/java/org/keycloak/validation/DefaultClientValidationProvider.java

```diff
@@ -58,7 +59,22 @@
         BACKCHANNEL_LOGOUT_URL("backchannelLogoutUrl",
                 "Backchannel logout URL is not a valid URL", "backchannelLogoutUrlIsInvalid",
                 null, null,
-                "Backchannel logout URL uses an illegal scheme", "backchannelLogoutUrlIllegalSchemeError");
+                "Backchannel logout URL uses an illegal scheme", "backchannelLogoutUrlIllegalSchemeError"),
+
+        LOGO_URI("logoUri",
```

Comment:
Does it make more sense to use constants from the "config wrapper"?

Reply:
I have created constants variables to ClientScopeModel. If we move to only OIDC PR, we could move them to OIDCConfigAttributes. I prefer to keep them in ClientScopeModel for SAML future extension.